bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -385,21 +366,22 @@ <h3 class="section-heading">{{ _('What we’re reading:') }}</h3>
           tag_label=_('Developers'),
           title=_('Are you a developer?'),
           ga_title='Are you a developer?',
-          desc=_('Get the latest features, fast performance and the development tools you need to build for the open web with Firefox Develop Edition.'),
-          image_url='home/2018/cards/nov/developer.png',
+          desc=_('Get the latest features, fast performance and the development tools you need to build for the open web with Firefox Developer Edition.'),
+          image_url='home/2018/cards/dec/developer.png',
           aspect_ratio='mzp-has-aspect-16-9',
           link_url=url('firefox.developer.index'),
         )}}
         <!-- 17 -->
         {{ card(
           class='mzp-c-card-medium',
-          tag_label=_('Firefox'),
-          title=_('Get Firefox Focus'),
-          ga_title='Get Firefox Focus',
-          desc=_('The latest version comes with search suggestions, revamped visual design and an under-the-hood surprise for Android users.'),
-          image_url='home/2018/cards/nov/focus.png',
+          tag_label=_('Developers'),
+          title=_('Rust 2018 is here... but what is it?'),
+          ga_title='Rust 2018 is here but what is it?',
+          desc=_('With this edition, we’ve focused on productivity… on making Rust developers as productive as they can be.'),
+          image_url='home/2018/cards/dec/rust.png',
```

Comment:
The image here could do with a stroke around it to match the developer image?
<img width="1265" alt="screen shot 2018-12-17 at 05 37 40" src="https://user-images.githubusercontent.com/400117/50082014-e7bed200-01bd-11e9-9e6f-09f923e4950d.png">